@xen-orchestra/fs/src/_encryptor.js

```diff
@@ -2,7 +2,8 @@ const { pipeline } = require('node:stream')
 const { readChunk } = require('@vates/read-chunk')
 const crypto = require('crypto')
 
-export const DEFAULT_ENCRYPTION_ALGORITHM = 'aes-256-gcm'
```

Comment:
@fbeauchamp Don't you think there should be a list somewhere of supported algorithms?

I'm a bit ill-at-ease to not be able to see that `aes-256-gcm` is supported anymore.